packages/forms-web-app/src/controllers/selected-appeal/appeal-details/appeal-details-rows.js

```diff
@@ -19,26 +22,40 @@ const {
 exports.detailsRows = (caseData) => {
 	return [
 		{
-			keyText: 'Agent name',
-			valueText: formatAgentDetails(caseData),
-			condition: (caseData) => caseData.yourFirstName
+			keyText: 'Was the application made in your name?',
+			valueText: formatYesOrNo(caseData, 'isAppellant'),
+			condition: () => true
 		},
 		{
-			keyText: 'Named on the application',
-			valueText: `${caseData.appellantFirstName} ${caseData.appellantLastName}`,
-			condition: (caseData) => caseData.appellantFirstName
+			keyText: "Applicant's name",
+			valueText: formatApplicantDetails(caseData),
+			condition: (caseData) => !caseData.isAppellant
 		},
 		{
-			keyText: 'Application reference',
-			valueText: caseData.LPAApplicationReference,
-			condition: (caseData) => caseData.LPAApplicationReference
+			keyText: 'Contact details',
+			valueText: formatContactDetails(caseData),
+			condition: () => true
+		},
+		{
+			keyText: 'Phone number',
+			valueText: caseData.appellantPhoneNumber,
+			condition: (caseData) => caseData.appellantPhoneNumber
 		},
-
 		{
 			keyText: 'Site address',
 			valueText: formatAddress(caseData, '\n'),
 			condition: (caseData) => caseData.siteAddressLine1
 		},
+		{
+			keyText: 'What is the area of the appeal site?',
+			valueText: `${caseData.siteAreaSquareMeters}m<sup>2</sup>`,
+			condition: (caseData) => caseData.siteAreaSquareMeters
+		},
+		{
+			keyText: 'Is the site in a green belt',
+			valueText: 'Yes',
+			condition: (caseData) => caseData.appellantGreenBelt
```

Comment:
This is a bizarre design choice

Reply:
By design, not by you to be clear

Reply:
I am unsure whether the display logic will change, but it may well do I suppose.